airflow/models.py

```diff
@@ -3524,9 +3543,34 @@ def run(
             donot_pickle=configuration.getboolean('core', 'donot_pickle'),
             ignore_task_deps=False,
             ignore_first_depends_on_past=False,
-            pool=None):
+            pool=None,
+            delay_on_limit_secs=1.0):
```

Comment:
I'm not sure about making it a positional argument, it would need to be included in all other places where `BackfillJob` is already instantiated in the codebase, and also in the future. Furthermore, I'm not sure if it's strictly required, since the underlying code won't always use it, as long as there's a sensible default for when it is needed. 

Ideally you'd have a `defaults` module (I guess `settings` would be similar) in a top package where all these values can live in one place and just make a reference to everywhere else in the codebase if needed, given that we have the same issue with all other optional args in this function and others similar. I could change this to a similar approach.